notebooks/test_exgauss_wald.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>this looks good, at the very end we can clean this up a bit more, comment things and consider if it can be made part of the documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lnccbrown/ssm-simulators/pull/233/discussion/#comment-2507597869'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>